modules/auxiliary/dos/http/wordpress_directory_traversal_dos.rb

```diff
@@ -0,0 +1,160 @@
+##
+# This module requires Metasploit: http://www.metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  include Msf::Auxiliary::Dos
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name'            => 'WordPress Traversal Directory DoS',
+      'Description'     =>  %q{Cross-site request forgery (CSRF) vulnerability in the wp_ajax_update_plugin function in wp-admin/includes/ajax-actions.php in WordPress before 4.6 allows remote attackers to hijack the authentication of subscribers for /dev/random read operations by leveraging a late call to the check_ajax_referer function, a related issue to CVE-2016-6896.},
```

Comment:
can you break this up to multiple lines so its easier to read similar to https://github.com/rapid7/metasploit-framework/pull/8422/files#diff-992718cb1437d7e3d005df7439f53f30R14